README.md

```diff
@@ -80,12 +87,24 @@ global_logger(QXContexts.Logger.QXLoggerMPIPerRank())
 
 # Running with MPI
 
-MPI is used to use multiple processes for computation. The `mpiexecjl` script can be used to launch julia on multiple processes. See [MPI.jl documentation]() for details on how to set this up. For example to run the above example with 4 processes one would use the following:
+MPI is used to use multiple processes for computation. The `mpiexecjl` script can be used to launch Julia on multiple processes. See [MPI.jl documentation](https://juliaparallel.github.io/MPI.jl/latest/configuration/#Julia-wrapper-for-mpiexec) for details on how to set this up. For example to run the above example with 4 processes one would use the following:
 
 ```
 mpiexecjl --project -n 4 julia --project bin/qxrun.jl -d examples/ghz/ghz_5.qx -o examples/ghz/out.jld2
```

Comment:
Is there an extra `--project` here? or are they both needed?

Reply:
Good catch. Only one of these is required here and think it is the one to `mpiexecjl`. Will update accordingly.